source/content/php-versions.md

```diff
@@ -36,11 +36,11 @@ Changes made to the `pantheon.yml` file on a branch **are not** detected when cr
 | [7.4](https://v74-php-info.pantheonsite.io/) | ✅          | Active  |
 | [7.3](https://v73-php-info.pantheonsite.io/) | ❌          | EOL     |
 | [7.2](https://v72-php-info.pantheonsite.io/) | ❌          | EOL     |
-| [7.1](https://v71-php-info.pantheonsite.io/) | ❌          | EOL     |
-| [7.0](https://v70-php-info.pantheonsite.io/) | ❌          | EOL     |
+| [7.1](https://v71-php-info.pantheonsite.io/) | 🛑          | EOL     |
+| [7.0](https://v70-php-info.pantheonsite.io/) | 🛑          | EOL     |
```

Comment:
Why is 7.3 and 7.2 an X, but 7.1 and 7.0 is a stop sign? And then we go back to X again for end of sale, but add an !, unless it's 5.3 in which case it's "do not enter"?  I think this is too much icon variety. How about we have just three icons: active, EOL and EOS.

Reply:
I was including different examples of icons.  Nathan said he didn't like the stop signs (I don't either), so i'll remove those (even though this PR is just brainstorming)